### ISSUE TITLE
dynatrace_aws_credentials taking 10-20x longer to apply after 1.25 update

### ISSUE THREAD

**Issue by FinUD** [dynatrace-oss/terraform-provider-dynatrace#222]
**Describe the bug**
After updating to 1.25 to fix bug #220 we noticed that the apply times for our terraform plans that utilized the dynatrace_aws_credentials resource took roughly 30-50 minutes where they used to take around 2-3.

**To Reproduce**
1. Configure and apply monitored services using the dynatrace_aws_credentials resource using v1.24 of the provider.
2. Rerun on v1.25 and time the difference.

**Expected behavior**
It to take the same time or atleast not 10-20 times longer.

**Additional context**
We are applying 71 monitored services with a total of 2354 metrics across 39 accounts. Happy to provide the full list of services/metrics if that helpful.

**Comment by FinUD** [comment]
Thanks @Dynatrace-Reinhard-Pilz for the detailed response and the timely updates, this is fantastic. We greatly appreciate the work done in recent times on the provider and what's still to come.

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hello @FinUD ,

With the upcoming release you will be able to specify `supporting_services_managed_in_dynatrace = true` within the resource `dynatrace_aws_credentials`. As explained before, that instructs Terraform to disregard `supporting_services_to_monitor` blocks and instead leave them as they are configured within your Dynatrace environment. The performance impact is now again considerably low. 

If you wish to manage the supporting services via Terraform, you *are* still able to do that by omitting `supporting_services_managed_in_dynatrace = true`. In that case you just need to be aware of the fact, that the `supporting_services_to_monitor` blocks need to be sorted within your Terraform modules alphabetically by `name` attribute.
Because that can be quite challenging if a lot of services are to be managed I'd recommend to use the [export functionality](https://registry.terraform.io/providers/dynatrace-oss/dynatrace/latest/docs/guides/export-v2) here. It would produce a .tf file that's already pre-sorted for you when executing something like this:
```
terraform-provider-dynatrace_v1.26.0.exe -export -id -flat dynatrace_aws_credentials=<THE_ID_OF_THE_CREDENTIALS>
```

With the planned `dynatrace_aws_service` resource there won't be any sorting necessary anymore. As explained that one has to wait for a future release, it won't be included in v1.26.0 yet.

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Handled with [v1.32.0](https://github.com/dynatrace-oss/terraform-provider-dynatrace/releases/tag/v1.32.0)
